db/migrate/20181112182009_create_projects.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProjects < ActiveRecord::Migration[5.2]
+  def change
+    create_table :projects do |t|
+      t.string :title
+      t.string :created_by
```

Comment:
this looks strange (naming and string type), why not to use references or belongs to and name it user not created by?